front/pages/api/v1/w/[wId]/data_sources/[name]/databases/[dId]/tables/[tId]/rows/index.ts

```diff
@@ -146,8 +151,8 @@ async function handler(
         });
       }
 
-      const { rows, total } = listRes.value;
-      return res.status(200).json({ rows, offset, limit, total });
+      const { rows: rowsList, total } = listRes.value;
+      return res.status(200).json({ rows: rowsList, offset, limit, total });
```

Comment:
why the change?

Reply:
name clash with the `rows` variable in both the `GET` and `POST`